VisualPinball.Unity/VisualPinball.Unity/VPT/ItemBehavior.cs

```diff
@@ -121,6 +140,10 @@ protected virtual void Awake()
 
 		protected virtual void OnDrawGizmos()
 		{
+			if (_meshDirty) {
+				RebuildMeshes();
+			}
+

```

Comment:
Unrelated bugfix I assume?

Reply:
No actually, this is meant to cover the case where you undo a material change and we need to rebuild meshes for the affected items. We don't want to rely on the inspector windows themselves to handle it (they won't necessarily be visible after all), so instead we'll check for dirty when the scene views draw.

Reply:
Gotcha, thanks for the explanation!